drake/examples/Cars/simulateLCM.cpp

```diff
@@ -78,11 +89,22 @@ int main(int argc, char* argv[]) {
   // be reused
   DrakeJoint::FloatingBaseType floating_base_type = DrakeJoint::QUATERNION;
 
-
   auto rigid_body_sys = make_shared<RigidBodySystem>();
-  rigid_body_sys->addRobotFromFile(argv[1], floating_base_type);
-  auto const & tree = rigid_body_sys->getRigidBodyTree();
-  for (int i=2; i<argc; i++)
+
+  // The following variable, weld_to_frame, is only needed if the model is a
+  // URDF file. It is needed since URDF does not specify the location and
+  // orientation of the car's root node in the world. If the model is an SDF,
+  // weld_to_frame is ignored by the parser.
```

Comment:
thinking about this more, this seems like a confusing api design.  if i pass a frame into the addRobotFromFile, i would expect it to change the root location.  for sdf, which can also specify a non-zero location for the first joint, this would be an additional offset.  but urdf, it might be the only offset.